### PR TITLE
prepare to release patch of embroider/compat

### PR DESCRIPTION
**Pull request by mansona**

### CODE REVIEW

**Review by NullVoxPopuli — APPROVED** [review]

File: .release-plan.json [hunk 1]
```diff
@@ -10,103 +10,31 @@
       "oldVersion": "3.0.0"
```
Comment (NullVoxPopuli) [inline .release-plan.json:10]: I'm confused by the diff in this file -- were its contents stale?

Comment (NullVoxPopuli) [reply .release-plan.json:10]: if the contents are stale, should we clear the config for each package after its release?

**Review by ef4 — APPROVED** [review]

**Comment by mansona** [comment]
fyi I'm going to merge in spite of the flaky tests. Github seems to be stuck with a queued runner 😫

File: .release-plan.json (hunk 1 above)
Comment (void-mAlex) [reply .release-plan.json:10]: would also work better when publishing bugfixes in lower versions

**Pull request merged**